Makefile

```diff
@@ -1,8 +1,11 @@
 build:
 	rm -rf ./dist
-	./node_modules/.bin/babel src --config-file ./babel.config.json --out-dir dist --source-maps --ignore **/*.test.jsx,**/__mocks__,**/__snapshots__,**/setupTest.js --copy-files
+	tsc --emitDeclarationOnly
+	./node_modules/.bin/babel src --config-file ./babel.config.json --out-dir dist --source-maps --ignore **/*.test.jsx,**/*.test.tsx,**/__mocks__,**/__snapshots__,**/setupTest.js --copy-files --extensions ".tsx,.jsx"
 	# --copy-files will bring in everything else that wasn't processed by babel. Remove what we don't want.
-	rm -rf dist/**/*.test.jsx
```

Comment:
This didn't work as expected I think, build still had tests files inside `tests` folder (e.g. `Datatable/tests/` is still in the `dist` directory with tests files in there), I think it deleted files only one level deep not going into all subdirectories.

Rewrote this to make sure we delete all tests and md files from the build